src/main.cc

```diff
@@ -808,12 +805,6 @@ serverConnectionsOpen(void)
         asnInit();
```

Comment:
`mgr:asndb` report is similarly broken, but I am not fixing that report in this PR because asnInit() call timing requirements/preconditions are more complex, and because ASN code itself has serious problems that I would rather not get entangled with right now. We will come back to that code in the near future.